docs/book/src/user/quick-start.md

```diff
@@ -0,0 +1,477 @@
+# Quick Start
+
+In this tutorial we'll cover the basics of how to use Cluster API to create one or more Kubernetes clusters.
+
+## Prerequisites
+
+- Install and setup [kubectl] in your local environment.
+- A Kubernetes cluster to be used as [management cluster](../reference/glossary.md#management-cluster), for the purpose of this quick start, we suggest you to use [kind].
+    ```bash
+    kind create cluster --name=clusterapi
+    export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+    ```
+
+## Installation
+
+Using [kubectl], let's create the components on the management cluster:
+
+#### Install Cluster API Components
+
+Check the [releases](https://github.com/kubernetes-sigs/cluster-api/releases) for an up-to-date components file.
+
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.2.1/cluster-api-components.yaml
+```
+
+#### Install the Bootstrap Provider Components
+
+{{#tabs name:"tab-installation-bootstrap" tabs:"Kubeadm"}}
+{{#tab Kubeadm}}
+
+Check the [Kubeadm provider releases](https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/releases) for an up-to-date components file.
+
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/releases/download/v0.1.0/bootstrap-components.yaml
+```
+
+{{#/tab }}
+{{#/tabs }}
+
+
+#### Install Infrastructure Provider Components
+
+{{#tabs name:"tab-installation-infrastructure" tabs:"AWS,vSphere"}}
+{{#tab AWS}}
+
+Check the [AWS provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases) for an up-to-date components file.
+
+For more information about prerequisites, credentials management, and or permissions for AWS, visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/docs/getting-started.md).
+
+```bash
+kubectl create -f https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v0.4.0/infrastructure-components.yaml
```

Comment:
```suggestion
kubectl create -f https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v0.4.1/infrastructure-components.yaml
```

I suspect we will not release v0.4.0 similar to how we handled the v0.2.0 release for capi.